daml-lf/validation/src/main/scala/com/digitalasset/daml/lf/validation/Serializability.scala

```diff
@@ -62,7 +62,9 @@ private[validation] object Serializability {
         checkType(targ)
       case TBuiltin(builtinType) =>
         builtinType match {
-          case BTInt64 | BTText | BTTimestamp | BTDate | BTParty | BTBool | BTUnit =>
+          case BTInt64 | BTText | BTTimestamp | BTDate | BTParty | BTBool | BTUnit |
+              BTAnyException | BTGeneralError | BTArithmeticError | BTContractError =>
```

Comment:
I think we decided `AnyException` is not serializable for now?

Reply:
Sure.

Reply:
This should be fixed. In Haskell the `AnyException` raises `URAnyException`.